modules/@apostrophecms/cache/index.js

```diff
@@ -0,0 +1,114 @@
+// A general purpose cache implementation for improved performance in all areas
```

Comment:
Could this be an a different PR where it could be more of the focus?

Reply:
Or was this previously in another file?

Reply:
This was part of the decision taken to rename the module to the singular, while fixing its API so that, among other things, that would not be super confusing.

Reply:
Also it's contained to one file and its test file, so I don't think it greatly burdens the PR. 🙏